posthog/rate_limit.py

```diff
@@ -27,7 +28,7 @@ class PassThroughSustainedRateThrottle(PassThroughThrottle):
 
 
 class DestroyClickhouseModelThrottle(UserRateThrottle):
-    rate = "10/hour"
+    rate = get_instance_setting("DESTROY_CLICKHOUSE_MODEL_THROTTLE_RATE")
```

Comment:
Doing things this way _always_ causes postgres queries as the app is booting, not when rate limiting is used.

We should avoid this, but I'm looking into fixing the endpoint first.